<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KVirtualStorageView</class>
 <widget class="QWidget" name="KVirtualStorageView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="5">
    <widget class="KUrlRequester" name="kurlrequester_file"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_type">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox_type">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Disk</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CD-Rom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Floppy</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_interface">
     <property name="text">
      <string>Interface</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="comboBox_interface">
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ide</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">scsi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">sd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">mtd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">floppy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">pflash</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">virtio</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_file">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="5">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="5">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
